iam-login-service/src/main/resources/i18n/sv/messages.json

```diff
@@ -25,7 +25,6 @@
             "access-token-no-timeout": "Access-token har oändlig livslängd",
             "acr-values": "Default ACR-värden",
             "acr-values-placeholder": "nytt ACR-värde",
-            "token-timeout-help": "Ange tiden i sekunder, timmar eller minuter.",
```

Comment:
Duplicate entry error "token-timeout-help"
See line 153. I removed the unsorted one.